### PR TITLE
Feature/dx 1626 design adjustments

### PR DESCRIPTION
**Pull request by trieutran91**

### CODE REVIEW

File: _includes/github.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% assign github_branch = site.github.branch | default : "develop" %}
 
 <a href="{{ site.github.repository_url }}/tree/{{ github_branch }}/{{ page.path }}"
-    class="github" target="_blank"
+    id="github" class="github" target="_blank"
```
Comment (asbjornu) [inline _includes/github.html:4]: If we add `id="github"` I suppose we can remove `class="github"`?

```suggestion
    id="github" target="_blank"
```

File: _sass/title-header.scss [hunk 2]
```diff
@@ -49,6 +49,12 @@
     display: flex;
     justify-content: center;
 
+    .github {
+        path {
+            fill: $black;
+        }
+    }
```
Comment (asbjornu) [inline _sass/title-header.scss:56]: Just cram this into the `#github` selector instead?

```suggestion
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by trieutran91** [comment]
These changes allows us to use the Github logo with white background and black cat, and the inverse.
The logo with white background works better on a dark background such as https://developer.swedbankpay.com/payment-instruments/.
While the logo with black background works on the pages with the new design such as https://developer.stage.swedbankpay.com/.

Can this work until we have worked out a design for the whole page?

**Comment by asbjornu** [comment]
Hm. I'm not sure that we can maintain both black on white and white on black octocats simultaneously. When the theme is upgraded, all pages on the Developer Portal will make use of the new theme, it can't be conditioned. So it's an all or nothing approach where we either make it work black on white or white on black. Unless there's something I've missed?

**Comment by trieutran91** [comment]
Well, we have a different design on the front page/Checkout v3 and the rest of the portal. 
Front page/Checkout v3 have a white header, while the other pages have the dark brown header ref https://developer.stage.swedbankpay.com/checkout-v2/payment-menu.
I believe that one of the arguments behind that decision was to separate the new Checkout v3 somehow. But yeah, it is not adequate.

**Comment by asbjornu** [comment]
Hm, I think we're both right somehow. We don't need both `class="github"` and `id="github"`, but the placement of the different selectors you've added and modified is correct, since they then apply to different parts of the portal. I think if you just change the `.github` selector to `#github` and remove `class="github"`, it should work. But please test it.

**Comment by trieutran91** [comment]
> Hm, I think we're both right somehow. We don't need both `class="github"` and `id="github"`, but the placement of the different selectors you've added and modified is correct, since they then apply to different parts of the portal. I think if you just change the `.github` selector to `#github` and remove `class="github"`, it should work. But please test it.

It worked! I did not manage to wrap my head around that the placement of the selectors where enough to apply for different parts. No need for `class="github"` indeed!

**Review by asbjornu — APPROVED** [review]

**Pull request merged**